gaitutils/videos.py

```diff
@@ -55,12 +56,39 @@ def convert_videos(vidfiles, check_only=False):
     vidconv_opts = cfg.general.videoconv_opts
     if not (vidconv_bin.is_file() and os.access(vidconv_bin, os.X_OK)):
         raise RuntimeError(f'Invalid configured video converter: {vidconv_bin}')
-    procs = []
+
+    procs = []    
+
     for vidfile in convfiles:
-        cmd = [vidconv_bin] + vidconv_opts.split() + [vidfile]
-        # supply NO_WINDOW flag to prevent opening of consoles
-        p = subprocess.Popen(cmd, stdout=None, creationflags=0x08000000)
+        if vidconv_opts == '':
+            """ For compatibility purposes, if no video converter options are
```

Comment:
I think the Python convention is that you should write "normal" comments using `#`, while the triple quoted strings are for docstrings only. See e.g. https://peps.python.org/pep-0008/#block-comments
At least for me, when I see a string like this, I immediately start looking for the preceding function.